entries/p/pixel.labcorp.com.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Pixel by Lapcorp": {
+    "domain": "pixel.labcorp.com",
+    "img": "pixel.labcorp.com.svg",
```

Comment:
```suggestion
```

The `img` field is only necessary when the image's file name is different to `domain`.svg